Tests/SentryTests/SentrySpanContextTests.swift

```diff
@@ -60,6 +60,32 @@ class SentrySpanContextTests: XCTestCase {
         XCTAssertEqual(data["status"] as? String, "ok")
     }
     
+    func testSampledNoSerialization() {
+        let id = SentryId()
+        let spanId = SpanId()
+        let parentId = SpanId()
+        
+        let spanContext = SpanContext(trace: id, spanId: spanId, parentId: parentId, operation: someOperation, sampled: .no)
+        spanContext.status = .ok
+        
+        let data = spanContext.serialize()
+        
+        XCTAssertEqual(data["sampled"] as? String, "false")
```

Comment:
`l`: Can we please also add a test for `.yes` / `true`.

Reply:
We have in line 58.